gobblin-docs/user-guide/Gobblin-CLI.md

```diff
@@ -28,29 +28,29 @@ Gobblin ingestion applications
 
 Gobblin ingestion applications can be accessed through the command `run`:
 ```bash
-bin/gobblin run [listQuickApps] [<quick-app>] -jobName <jobName> [OPTIONS]
+bin/gobblin cli run [listQuickApps] [<quick-app>] -jobName <jobName> [OPTIONS]
```

Comment:
@autumnust, with this change, I am targetting to combine all services and commands together because the config and other mgmt is the same for all services and commands. since the gobblin is just link to gobblin.sh, with this change it will handle the cli as a command. so following becomes the new way of using gobblin script for various activities.
`gobblin.sh  <command> <params>
gobblin.sh  <service-name> <start|stop|status>

commands values: admin, cli, statestore-check, statestore-clean, historystore-manager
service values: standalone, cluster-master, cluster-worker, aws, yarn, mr, service`

Reply:
@autumnust , does that make sense ?

Reply:
Sorry for getting back to this late. 
Yes that makes sense to me. I see them in `gobblin.sh` printUsage() method. Can you add/edit documentation to mention the existence of `gobblin.sh` so that newer users will be aware of it ?

Reply:
ok sure, it requires log of doc changes, and some reorganization, which i can take care of but can we get #2586 merged? otherwise i ll have lot of conflicts.

Reply:
@jhsenjaliya Gobblin CLI is already an aggregator for functionality, no reason to add an additional `cli` to the command. Today there is `bin/gobblin run` which should stay as is, additionally there can be `bin/gobblin admin`, etc.

Reply:
Hi @ibuenros ,  the `cli` has lot of options other than `run`, like `passwordManager`, `watermarks`, etc...  so we can keep it under `cli` category for now, and in future, those options can be added directly to the `gobblin.sh` if we really want to have `run` command directly under gobblin  without `cli`.
here the problem I m trying to solve is mostly inconsistency and lack of clarity among functionality of all gobblin scripts. I have also added more detail GOBBLIN-707 jira ticket to explain why we should combine all these scripts.

Reply:
My point is that there is no reason to move all of those applications down one level. Instead, everything else that you would want to add to the `bin/gobblin` command should be implemented as a `CliApplication`.

I do see the benefit of aggregating all functionality, I just think it should be done without changing current commands.

Reply:
ok , sure.
is `gobblin admin <args>` fine ?
is it only about `gobblin cli run <args>`? then i can bring the `run`, `watermarks`, `passwrodManager`, etc commands to gobblin.sh level so that the command signature wont change from existing one.

Reply:
What is the proposal on how to do that though? Actually I'm a bit confused with this PR, there are a lot of removed shell scripts, but there seems to be no replacement.

Reply:
to handle it on bash script level and call the `GobblinCli` with appropriate args for each command accordingly.
btw, each removed shell script has support to execute via this updated `gobblin.sh`.
if you can pull this PR and try some of the commands, it will be clear on how things are working with these updates. git diff is bit confusing for such large PR. Thanks

Reply:
@ibuenros , lets take this offline over email. Thanks

Reply:
Hi @ibuenros, I have added the mapping of old gobblin script to new gobblin command in jira ticket and also started email thread: "GOBBLIN-707 Review", would you please take a look and reply there. Thanks